src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.Context.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.FlowAnalysis
+{
+    internal sealed partial class ControlFlowGraphBuilder
+    {
+        internal struct Context
```

Comment:
>Context [](start = 24, length = 7)

Although this is an internal type, some comment/description would be useful. What's the purpose and difference with `ImplicitInstanceInfo` (which contains similar information)?